src/Illuminate/Database/Query/Builder.php

```diff
@@ -1613,7 +1613,7 @@ public function sum($column)
 	 * Retrieve the average of the values of a given column.
 	 *
 	 * @param  string  $column
-	 * @return mixed
+	 * @return float|int
```

Comment:
Makes sense for an average? I'd even say you can do "float" for all of them. Even though PHP may or may not turn some of the values into integers, I don't see a reason why we cannot simply treat them as floats.